sheet_to_triples/field.py

```diff
@@ -101,8 +101,8 @@ def as_type(self):
         return _must(_TYPES.get(_str(self._value)[:4].lower()), TypeError)
 
     @property
-    def as_type_prefix(self):
```

Comment:
There's questions about how much we care about actively maintaining transforms that use properties that we change like this. I see we're only using this in `mgsbr_points` in `transforms` - I can foresee us trying to run that transform again, having it fail, and wondering why, but also wonder how likely we are to actually run that transform again ever.

Reply:
I bundled a change to that transform along with the new transform that wanted this more flexible spelling.

In general I think we should aim to keep old transforms valid at least, though they might not pick up new spellings for things,